src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -149,26 +184,33 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
                     rows={5}
                   />
 
-                  <div className="notification notification-red" data-testid="statusFormWarning">
+                  <Notification warning data-testid="statusFormWarning">
                     {warning}
-                  </div>
-                  <div className="notification notification-red" data-testid="statusFormError">
-                    {error && error.response && error.response.status === 403 ? 'Je bent niet geautoriseerd om dit te doen.' : '' }
-                    {error && error.response && error.response.status !== 403 ? 'De gekozen status is niet mogelijk in deze situatie.' : '' }
-                  </div>
+                  </Notification>
+
+                  <Notification warning data-testid="statusFormError">
+                    {error && error.response && error.response.status === 403
```

Comment:
Could we turn these codes into constants? `HTTP_FORBIDDEN`

Could also replace conditional (?) by short-circuit evaluation. Removing the need for the empty string.

Reply:
Good point; I'll make a ticket for the constants suggestion.